AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -11184,6 +11184,11 @@ forum.gamer.com.tr#?#.p-body-inner > .notices > .notice:has(> .notice-content >
 bethesda.net#?#visor-alert > div > div > div:last-child:has(> a[href$="/cookie-policy"])
 maximum.com.tr#?#.sticky_cookie_warning:has(> a[href^="/TR/gizlilik-politikamiz/Sayfalar/gizlilik-politikamiz.aspx"])
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176447
+||laroche-posay.ru/static/*/js/cookies-popup/
+laroche-posay.ru#$#body { overflow: auto !important; }
```

Comment:
Directives are required, because in CB for Samsung Browser the CSS rule will not be applied and scrolling will be blocked.

Just add
```
! https://github.com/AdguardTeam/AdguardFilters/issues/176447
!#if (!adguard_app_ios && !adguard_ext_android_cb)
||laroche-posay.ru/static/*/js/cookies-popup/
laroche-posay.ru#$#body { overflow: auto !important; }
!#endif
```